tests/integration/testlib/bondlib.py

```diff
@@ -28,6 +28,17 @@
 
 @contextmanager
 def bond_interface(name, slaves, extra_iface_state=None):
+    desired_state = generate_create_bond_state(name, slaves, extra_iface_state)
+
+    libnmstate.apply(desired_state)
+
+    try:
+        yield desired_state
+    finally:
+        libnmstate.apply(generate_remove_bond_state(name), verify_change=False)
+
+
+def generate_create_bond_state(name, slaves, extra_iface_state=None):
```

Comment:
I find the usage of two verbs here problematic (generate_create).
`create_bond_state` and `remove_bond_state` is clear enough.

Reply:
No longer applies.